src/site/stages/build/plugins/create-environment-filter.js

```diff
@@ -8,6 +8,14 @@ function createEnvironmentFilter(options) {
     for (const fileName of Object.keys(files)) {
       const file = files[fileName];
 
+      // Do not include CLPs on production.
+      if (
+        environmentName === ENVIRONMENTS.VAGOVPROD &&
+        file.entityBundle === 'campaign_landing_page'
+      ) {
+        delete files[fileName];
+      }
+
```

Comment:
i _think_ if this executed by the preview server then `options.isPreviewServer` will be `true`

Reply:
ah perfect, thanks!